Makefile

```diff
@@ -42,9 +43,23 @@ nupkg: lib lib-win
 	#dotnet nuget push *.nupkg --api-key <API_KEY> --source 	https://api.nuget.org/v3/index.json
 
 .PHONY: deps
+.ONESHELL:
 deps:
+	# install gcc
 	sudo apt install gcc
 	sudo apt install gcc-mingw-w64
+	# install pcre
+	sudo apt install libpcre3-dev
+	# install swig
+	wget https://downloads.sourceforge.net/swig/swig-$(SWIG_VERSION).tar.gz
+	mkdir -p swig && tar -xzvf swig-$(SWIG_VERSION).tar.gz -C swig --strip-components 1
+	cd swig
+	./configure; sudo make; sudo make install
+	cd ..; rm -rf swig; rm -f swig-$(SWIG_VERSION).tar.gz
+	echo "export SWIG_PATH=usr/local/share/swig/bin" | sudo tee -a /etc/profile
+	echo "export PATH=$(SWIG_PATH):$(PATH)" | sudo tee -a /etc/profile
+	source /etc/profile
```

Comment:
Here we could create (and later clean up) a temporary directory in which to extract this.

This way `make deps` will not fail if we ever decide to have a directory named swig in the repo.

Reply:
I will rename it `temp` then. Feel free to come up with other suggestions if you think this is not ok.